WhaleSpotting/Controllers/SightingsController.cs

```diff
@@ -48,5 +48,11 @@ public async Task<ActionResult<SightingResponseModel>> ConfirmSighting([FromRout
             var sighting = await _sightings.ConfirmSighting(id);
             return sighting == null ? NotFound() : sighting;
         }
+
+        [HttpGet("localspecies")]
+        public async Task<IEnumerable<string>> GetSpeciesByCoordinates([FromQuery] string latitude, string longitude)
```

Comment:
```suggestion
        public async Task<IEnumerable<string>> GetSpeciesByCoordinates([FromQuery] string latitude, string longitude)
```
```suggestion
        public async Task<IEnumerable<string>> GetSpeciesByCoordinates([FromQuery] double latitude, double longitude)
```
We should be able to cast these here and can we have this return the actual enum not the string, then in the frontend we can say that it is that enum value as well

Reply:
Addressed.